libs/solid/quiz/src/lib/components/ranking-question/ranking-question.component.ts

```diff
@@ -0,0 +1,49 @@
+import { CdkDragDrop, moveItemInArray } from '@angular/cdk/drag-drop';
+import { Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges } from '@angular/core';
+import { QuizQuestion } from '../../state/quiz.model';
+
+@Component({
+  selector: 'solid-quiz-ranking-question',
+  templateUrl: './ranking-question.component.html',
+  styleUrls: ['./ranking-question.component.scss'],
+})
+export class RankingQuestionComponent implements OnInit,OnChanges {
+  @Input() public question!: QuizQuestion;
+  @Output() public nextQuestionClicked = new EventEmitter<boolean>();
+  
+  public SelectedAnswers!: number[];
```

Comment:
Nur eine kleine Frage/Anmerkung: haben wir eine Konvention, wann der erste Buchstabe einer Variablen großgeschrieben wird?

Ich würde sagen, dass alle Variablen klein anfangen und CamelCase verwendet wird. Aber es ist vielleicht auch nicht konsistent so. Ich würde es aber gut finden, wenn wir alle klein anfangen lassen WENN es keine sonstige Regel gibt.

Reply:
Die großgeschriebene Variablen wurden von dem vorherigen Entwickler definiert. Ich weiß auch nicht welche Konvention hier verwendet wird. 

Aber ich bleibe auch gerne mit den kleinen anfangenden Variablen

Reply:
Tatsächlich... 😅 
Hm - ich sehe keinen richtigen Grund dafür. Es gibt so ein paar Sachen, z.B. besondere Variablen, bei denen das evtl. sinnvoll ist, weil sie speziell sind. z.B. in den models, wenn die Daten so direkt aus dem Backend kommen. Aber das machen wir ja ansonsten auch nicht so. 

Aber gut - dann schlage ich vor, dass wir das bei allen neuen Variablen konsistent machen: Klein mit CamelCase.

Und wenn sowieso an einer Komponente gearbeitet wird, können wir es auch gleich einheitlich machen.

Reply:
wir.. Du... 😉